src/test/java/org/folio/rest/api/ItemEffectiveCallNumberTest.java

```diff
@@ -10,22 +13,29 @@
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
-
+import org.apache.commons.lang3.StringUtils;
 import org.folio.rest.jaxrs.model.EffectiveCallNumberComponents;
 import org.folio.rest.jaxrs.model.HoldingsRecord;
 import org.folio.rest.jaxrs.model.Item;
 import org.folio.rest.persist.PostgresClient;
+import org.folio.rest.support.builders.ItemRequestBuilder;
 import org.folio.util.ResourceUtil;
 import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+
 import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
 import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
 
 @RunWith(JUnitParamsRunner.class)
 public class ItemEffectiveCallNumberTest extends TestBaseWithInventoryUtil {
```

Comment:
What is the difference between this class and the `ItemEffectiveCallNumberComponentsTest`?

Reply:
The `ItemEffectiveCallNumberTest ` it is test for the migration script, while `ItemEffectiveCallNumberComponentsTest` it is test for the update logic itself.

Reply:
Could we rename `ItemEffectiveCallNumberTest` to something that reflect that it is for the migration script, e.g. `ItemEffectiveCallNumberDataUpgradeTest`?